lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -276,6 +178,27 @@ def host_reachable?(host, what = "Server")
         say("Succeeded.")
         true
       end
+
+      def cp_template(file, src_dir, dest_dir)
+        src_path = path_join(src_dir, file)
+        if src_path.to_s.include?(".erb")
+          dest_path = path_join(dest_dir, file.gsub(".erb", ""))
+          File.write(dest_path, ERB.new(File.read(src_path), nil, '-').result(binding))
+        else
+          FileUtils.cp src_path, dest_dir
+        end
+      end
+
+      def rm_file(file, dir)
```

Comment:
did you need this?

for some reason when reading `unconfigure_httpd_application`, I keep looking into this function to see if anything special is in here?

alternative is:
```ruby
File.rm(HTTPD_CONFIG_DIR.join(CONFIG_FILE[:external_auth]))
```
or
```ruby
File.rm HTTPD_CONFIG_DIR.join(EXTERNAL_AUTH_FILE)
```